### DISCUSSION TITLE
Processor: F2MC-16LX support

### DISCUSSION THREAD

**Discussion by mehmooda** [NationalSecurityAgency/ghidra#5406] [Show and tell]
I've created a new ghidra processor for the F2MC-16LX.

It can currently disassemble the instruction set

But only has partial pcode 

It's probably not ready to be included upstream
Just adding this issue as a notice if anybody wants to use/improve it 

https://github.com/mehmooda/F2MC-16LX

**Comment by mumbel** [comment]
@pRoxxxDD (https://github.com/NationalSecurityAgency/ghidra/discussions/3868)

**Comment by WilliamVanHUB** [comment]
@mehmooda I try to reverse engineer a F2MC-16LX MB90F352S with your processor but there are not any functions decoded. Can you explain how this works with Ghidra and your processor addition?
